lerna.json

```diff
@@ -1,6 +1,4 @@
 {
   "npmClient": "yarn",
-  "useWorkspaces": true,
-  "packages": ["packages/*"],
```

Comment:
This would now cause lerna to only version a subset of packages (only the `@grafana/` packages) which breaks versioning. Removing seems to solve things.